mindsdb/integrations/handlers/vertex_handler/vertex_client.py

```diff
@@ -1,16 +1,23 @@
 from mindsdb.utilities import log
 from google.cloud import aiplatform
-from google.oauth2 import service_account
 import pandas as pd
 
+from mindsdb.integrations.handlers.utilities.auth_utilities import GoogleServiceAccountOAuth2Manager
+
 logger = log.getLogger(__name__)
 
 
 class VertexClient:
     """A class to interact with Vertex AI"""
 
-    def __init__(self, credentials_info, args_json):
-        credentials = service_account.Credentials.from_service_account_info(credentials_info)
+    def __init__(self, credentials_url, credentials_file, credentials_json, args_json):
```

Comment:
looks like all 3 methods to pass credentials required (should be passed all of them)
but in reality only one
maybe to add default values them?